src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -7527,9 +7456,9 @@ void CalcHeatBalanceOutsideSurf(EnergyPlusData &state,
 
 Real64 GetQdotConvOutPerArea(EnergyPlusData &state, int const SurfNum)
 {
-    auto &Surface = state.dataSurface->Surface;
-    int OPtr = Surface(SurfNum).OSCMPtr;
-    if (Surface(SurfNum).OSCMPtr > 0) { // Optr is set above in this case, use OSCM boundary data
+    auto const &surface = state.dataSurface->Surface(SurfNum);
+    int OPtr = surface.OSCMPtr;
+    if (surface.OSCMPtr > 0) { // Optr is set above in this case, use OSCM boundary data
```

Comment:
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:7530]:(style),[constVariable],Variable 'Surface' can be declared as reference to const